yourgov/components/StaffMembersTable/ActiveFilters.tsx

```diff
@@ -0,0 +1,104 @@
+import { Text } from '@ag.ds-next/react/src/text';
+import { Flex } from '@ag.ds-next/react/src/flex';
+import { Button } from '@ag.ds-next/react/src/button';
+import { CloseIcon } from '@ag.ds-next/react/src/icon';
+import { Tags } from '@ag.ds-next/react/src/tags';
+import { GetDataFilters } from './lib/types';
+import { useSortAndFilterContext } from './lib/contexts';
+import { STATUS_MAP } from './lib/utils';
+
+type Entries<T> = {
+	[K in keyof T]: [K, T[K]];
+}[keyof T][];
+
+const getTagsFromFilters = ({
+	filters,
+	removeFilter,
+}: {
+	filters: GetDataFilters;
+	removeFilter: (filterKey: keyof GetDataFilters) => void;
+}) => {
+	const filterEntries = Object.entries(filters) as Entries<GetDataFilters>;
+
+	const tags: {
+		label: string;
+		onRemove: () => void;
+	}[] = [];
+
+	for (const [key, value] of filterEntries) {
+		if (!value) continue;
+		const formattedKey = formatFilterKey(key);
+		const onRemove = () => removeFilter(key);
+
+		if (key === 'activeUsers') {
+			if (!value) continue;
+			tags.push({
+				label: 'Active users',
+				onRemove,
+			});
+			continue;
+		}
+
+		if (key === 'status') {
+			tags.push({
+				label: `${formattedKey}: ${STATUS_MAP[value].label}`,
+				onRemove,
+			});
+			continue;
+		}
+
+		if (Array.isArray(value)) {
+			if (value.length === 0) continue;
+
+			tags.push({
+				label: `${formattedKey}: ${value.join(', ')}`,
+				onRemove,
+			});
+			continue;
+		}
+
+		tags.push({ label: `${formattedKey}: ${value}`, onRemove });
+	}
+
+	return tags;
+};
+
+function formatFilterKey(key: keyof GetDataFilters) {
+	return {
+		activeUsers: false,
+		dateJoinedFrom: 'Date joined',
```

Comment:
When you come to do the dates, you'll need to do the same as in the audit branch where if you only pick one datem it says "any" as a replacement https://github.com/agriculturegovau/agds-next/pull/1726/files#diff-dd543a065df2526c33bee34e2c2e5ca7c62c9aafb75505f63cd4fe2f48958995R67 https://design-system.agriculture.gov.au/pr-preview/pr-1726/storybook/iframe.html?args=&id=patterns-search-filters--table-medium&viewMode=story

Reply:
If this absolutely needs to happen I'll circle back around. I'd set it up as a flatter `lastActiveTo` and `lastActiveFrom` so two tags are created which means the input and tag match. i.e. in that link there is no "any" in the input. Happy to come back around but it'll be later on.

Reply:
No worries. I kinda like your approach. Would be good to flag with PM